clouddq/classes/dq_configs_cache.py

```diff
@@ -440,10 +442,62 @@ def _resolve_dataplex_entity_uri(
             )
         return clouddq_entity
 
+    def is_dataplex_entity(
+        self,
+        entity_uri: dq_entity_uri.EntityUri,
+        client: clouddq_dataplex.CloudDqDataplexClient,
+    ):
+        required_arguments = ["projects", "lakes", "locations", "zones"]
+        for argument in required_arguments:
+            uri_argument = entity_uri.get_configs(argument)
+            if not uri_argument:
+                logger.info(
+                    f"Failed to retrieve default Dataplex '{argument}' for "
+                    f"entity_uri: {entity_uri.complete_uri_string}. \n"
+                    f"'{argument}' is a required argument to look-up metadata for the entity_uri "
+                    "using Dataplex Metadata API.\n"
+                    "Ensure the BigQuery dataset containing this table "
+                    "is registered as an asset in Dataplex.\n"
```

Comment:
"registered as" -> "attached to"

Reply:
updated this to=> Ensure the BigQuery dataset containing this table is attached as an asset in Dataplex